dev-docs/lucene-upgrade.md

```diff
@@ -43,11 +43,7 @@ gradlew --write-locks
 ### `solr/licenses` update
 
 ```
-git rm lucene-*-9.3.0.jar.sha1
-
-# ???manually get new .sha1 files???
-
-git add lucene-*-9.4.0.jar.sha1
+gradlew updateLicenses
 ```
 
 ## Code
```

Comment:
Personally I use `./gradlew check -x test` for any changes without tests instead of just compile. I bet it's slower, but it runs all the spotless and other checks. 

Then when I'm happy it builds, I remove the `-x test` to run tests.

Not sure this makes it better in this case :)